### PR TITLE
Upgraded storage sample

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: storage/buckets.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+// [START create]
+/**
+ * Creates a new bucket with the given name.
+ *
+ * @param {string} name The name of the new bucket.
+ * @param {function} cb The callback function.
+ */
+function createBucketExample (name, callback) {
+  if (!name) {
```
Comment (jonparrott) [inline storage/buckets.js]: won't the client library handle this?

File: storage/buckets.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+ * @param {function} cb The callback function.
+ */
+function listBucketsExample (callback) {
+  // See https://googlecloudplatform.github.io/gcloud-node/#/docs/storage?method=getBuckets
+  storage.getBuckets(function (err, buckets, apiResponse) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Found ' + buckets.length + ' buckets!');
```
Comment (jonparrott) [inline storage/buckets.js]: print the list

File: storage/buckets.js (hunk 1 above)
Comment (jmdobry) [inline storage/buckets.js]: I think it does, don't know what came over me when I decided to check the argument in my sample function.

File: storage/buckets.js (hunk 2 above)
Comment (jmdobry) [inline storage/buckets.js]: The list will get printed when the user runs the sample. If I print it here it will get printed twice.

Comment (jonparrott) [inline storage/buckets.js]: Gotcha.

**Comment by jonparrott** [comment]
Looks (mostly) good to me.

**Comment by jmdobry** [comment]
Waiting on Travis

**Pull request merged**